app/views/team_mailer/team_registered_to_event.html.erb

```diff
@@ -2,5 +2,5 @@
 <p><%= t('.registered_to_event_info_html',
          team_link: link_to(@team.name, team_url(@team)),
          event_link: link_to(@event.name, event_url(@event))) %></p>
-<p><%= t('mailer.have_fun') %></p>
+<p><%= t('.have_fun') %></p>
```

Comment:
Nice find that we can use `.have_fun` without explicitly defining e. g. `user_added_to_team` as the locale to look into for the localization. 👍